src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--1">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
```

Comment:
```suggestion
      <div class="stars__star"></div>
    </div>
 
    <div class="stars stars--1">
      <div class="stars__star"></div>
      <div class="stars__star"></div>
```

checklist (but you can do not leave empty lines if the tag contains only 1 line)